io.github.dbchoco.Salawat.yml

```diff
@@ -0,0 +1,33 @@
+app-id: io.github.dbchoco.Salawat
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: salawat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: salawat
+    buildsystem: simple
+    build-commands:
+      - install --directory --mode=0755 /opt/Salawat
+      - install -Dm644 --target-directory=/opt/Salawat/ Salawat/*
+      - install -Dm755 Salawat/Salawat /app/bin/salawat
+      - install -Dm644 io.github.dbchoco.Salawat.png /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 io.github.dbchoco.Salawat.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 io.github.dbchoco.Salawat.metainfo.xml /app/share/metainfo/$FLATPAK_ID.appdata.xml
+    sources:
+      - type: archive
+        url: https://github.com/DBChoco/Salawat/releases/download/v1.0.0/Salawat-1.0-linux.tar.gz
```

Comment:
you need ti figure this out.

or better not depend on it use the JRE provided by the SDK so you don't have to limit your app to one architecture.

Reply:
this question remain unanswered.

Reply:
yeah sorry I didn't understand the question very well
for now I guess we could just limit the architectures until I can test it or until I can figure out a different way to build the app I guess ?

Reply:
it doesn't include the JRE. So it's unlikely to work.

Reply:
yes you're completely right, I missed the jre.
Do you think I'd be best to include them in the tar.gz or somehow install them through flatpak ?

Reply:
what I think is already in that thread. It involved using Flatpak provide Java SDK.

Reply:
I looked into the JDK extensions for flatpak and only found JDK 11 & JDK 17, however this app is made with Java 18, so I'd have compatibility issues

Reply:
> are you perhaps on aarch64 ? Because here I get another message. Or did you do it before the build finished maybe ?

as I mentionned previously, bundling the JDK would lead to these problem. Yes I was checking on aarch64 knowing it wouldn't work, because it's never the 'default'.